docs/cloud/pouta/images.md

```diff
@@ -46,41 +43,24 @@ keep the additional packages at a minimum.
 |Ubuntu-22.04    |**ubuntu**     |no  |
 |Ubuntu-24.04    |**ubuntu**     |no  |
 
-### CentOS-7
-CentOS is the community version of Red Hat Enterprise Linux (RHEL). CentOS-7
-is the distribution that the Pouta admins are the most accustomed to. 
-This is the second-newest major release of CentOS.
+### CentOS-9-Stream
 
-### CentOS-7-Cuda
-This is the same as CentOS-7 but comes with preinstalled NVIDIA drivers.
-These are useful in combination with the GPU flavors since you do not need
-to download the drivers yourself. We try to always have the latest NVIDIA
-drivers installed, but sometimes we are lagging a bit behind. These images
-are huge, so you should use the normal CentOS-7 image if you are not
-using a GPU-flavor VM.
-
-### CentOS-8-Stream and CentOS-9-Stream
-
-While CentOS-7 is still actively supported, [CentOS-8 has been declared end-of-life in December 2021](https://www.centos.org/centos-linux-eol/).
-The CentOS community is now actively maintaining CentOS-8-Stream and
-CentOS-9-Stream instead. The key differences lay in the relationship between
+The CentOS community is now actively maintaining CentOS-9-Stream. The key differences lay in the relationship between
```

Comment:
`The key differences lay in the relationship...` <- this sentence is still referring to our previous documentation, in which we were explaining the differences between CentOS-7 and CentOS-8-stream / CentOS-9-stream
I would suggest to remove the whole sentence (from `The key differences..` till `... new packages and updates, are deployed`. This means that you could just have something like:
```
The CentOS community is now actively maintaining CentOS-9-Stream, which is the upstream branch for RHEL...
```

Reply:
Fixed